index.html

```diff
@@ -148,349 +139,364 @@ <h2>Hello World! 😊</h2>
+                <div class="fish">
+                    <div class="body">
+                        <div class="eye"></div>
+                        <div class="fin-1"></div>
+                        <div class="fin-2"></div>
+                        <div class="fin-3"></div>
+                        <div class="fin-4"></div>
+                        <div class="tail"></div>
+                    </div>
+                    <div class="bubble">
+                        <div class="b-1"></div>
+                        <div class="b-2"></div>
+                    </div>
+                </div>
+            </div>
+            <p class="attribute">'made with ❤️ by Jonnie'</p>
         </div>
-      </div>
-    </div>
 
-    <!-- ********************************************************* -->
-    <!-- **********   ✔️  EDIT  HERE ✔️ *************************** -->
-    <!-- ********************************************************/ -->
-    <div class="card">
-      <div class="container">
-        <div class="roof">
-          <span class="roof-sheet roof-sheet-left"></span>
-          <span class="roof-sheet roof-sheet-right"></span>
-        </div>
-        <div class="triangle"></div>
-        <div class="house-body">
-          <div class="windows-top windows-container">
-            <div class="window">
-              <span class="window-glass"></span>
-              <span class="window-glass"></span>
-              <span class="window-glass"></span>
-              <span class="window-glass"></span>
-            </div>
-            <div class="window center-window">
-              <span class="window-glass"></span>
-              <span class="window-glass"></span>
-            </div>
-            <div class="window">
-              <span class="window-glass"></span>
-              <span class="window-glass"></span>
-              <span class="window-glass"></span>
-              <span class="window-glass"></span>
-            </div>
-          </div>
-          <div class="windows-container">
-            <div class="window">
-              <span class="window-glass"></span>
-              <span class="window-glass"></span>
-              <span class="window-glass"></span>
-              <span class="window-glass"></span>
-            </div>
-            <div class="window center-window">
-              <span class="window-glass"></span>
-              <span class="window-glass"></span>
-            </div>
-            <div class="window">
-              <span class="window-glass"></span>
-              <span class="window-glass"></span>
-              <span class="window-glass"></span>
-              <span class="window-glass"></span>
+        <div class="card h-100 shadow-sm">
+            <div class="text-center">
+                <div class="img-hover-zoom img-hover-zoom--colorize">
+                    <img class="shadow" src="./assets/images/logo.png" alt="Another Image zoom-on-hover effect" style="width: 100%;">
```

Comment:
Hey @M-Hidayatullah, our theme is CSS art and you've submitted an image. This is not allowed, moreover, it is our group's logo and your alt text is not describing the image at all. I'm gonna have to reject this and mark your PR as invalid